### PR TITLE
Fixes addNewService, same issue as #111

### PR DESCRIPTION
**Pull request by DBozhinovski**
Refactored SortableTabBar to a class instead of a stateless component. Saw that most of the code that had internationalization stuff inside was made up of stateful components anyway, so just tried to follow suit :)

Also added some placeholder strings for all the other languages, so no errors get logged. A chance for more hacktoberfest PRs, I suppose :)

Linter passes okay, tested it with most languages already and all seems in order.

### CODE REVIEW

File: src/i18n/locales/nl.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
   "infobar.buttonInstallUpdate": "Herstart & installeer update",
   "infobar.requiredRequestsFailed": "Kon services en gebruikerinformatie niet laden",
   "sidebar.settings": "Instellingen",
+  "sidebar.addNewService": "!!!Add new service",
```
Comment (mroeling) [inline src/i18n/locales/nl.json]: Voeg nieuwe service toe

File: src/components/services/tabs/Tabbar.js [hunk 2]
```diff
@@ -54,6 +54,8 @@ export default class TabBar extends Component {
       deleteService,
     } = this.props;
 
+    const { intl } = this.context;
```
Comment (adlk) [inline src/components/services/tabs/Tabbar.js]: We don't need `intl` here

File: src/components/services/tabs/Tabbar.js [hunk 3]
```diff
@@ -70,6 +72,7 @@ export default class TabBar extends Component {
           axis="y"
           lockAxis="y"
           helperClass="is-reordering"
+          intl={intl}
```
Comment (adlk) [inline src/components/services/tabs/Tabbar.js]: Please remove it from here as well

**Comment by DBozhinovski** [comment]
Updated with both.

**Comment by adlk** [comment]
PR merged. Thank you for your contribution!

**Pull request merged**